src/com/github/johnthagen/cppcheck/Configuration.java

```diff
@@ -18,7 +18,6 @@
             "Note: C++ projects should leave --language=c++ appended to the Cppcheck options to avoid some " +
                     "false positives in header files due to the fact that Cppcheck implicitly defaults to " +
                     "setting --language to \"c\" for .h files.\n\n" +
-                    "You should not include any --template={} in the options.\n\n" +
```

Comment:
Since we are using `--xml` now the specified `--template` parameter no longer matters.